apps/studio/schemas/organizer.ts

```diff
@@ -0,0 +1,34 @@
+import { HeartIcon } from "@sanity/icons";
+import { defineField, defineType } from "sanity";
+
+export default defineType({
+  name: "organizer",
+  icon: HeartIcon,
+  title: "Organizers",
+  type: "document",
+  fields: [
+    defineField({
+      name: "name",
+      title: "Name",
+      type: "string",
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+      name: "pronouns",
+      title: "Pronouns",
+      type: "string",
+    }),
+    defineField({
+      name: "linkedinURL",
+      title: "LinkedIn URL",
+      type: "url",
+      validation: (Rule) => Rule.required().uri({ scheme: ["http", "https"] }),
```

Comment:
We might not want to make this field required. We don't want the content team to add placeholder urls for people that don't have a LinkedIn or ones they haven't figured out yet.

Reply:
Good point, I'll make it optional then.

Reply:
Fixed in [babb28d](https://github.com/HackAtUCI/hack.ics.uci.edu/pull/12/commits/babb28de063249dfcad0d491a4467125cb60e11d)babb28d